python/ql/src/meta/ClassHierarchy/join-yml-files.py

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env python3
+
+"""Concerns were raised about performance on Windows with having 2.5 k files for modeling, and it was recommended we join them all together when shipping.
+
+This script does that.
+"""
+
+import sys
+import glob
+import os
+
+from shared_subclass_functions import *
+
+if joined_file.exists():
+    sys.exit(f"File {joined_file} already exists")
+
+package_data = gather_from_existing()
+as_lists = list()
+for data in package_data.values():
+    as_lists.extend(list(t) for t in data)
+as_lists.sort()
+
+
+to_write = wrap_in_template(as_lists)
+write_data(to_write, joined_file)
+
+print("Joined all files into", joined_file)
+
+for f in glob.glob(f"{subclass_capture_path}/auto-*.model.yml", recursive=True):
+    os.unlink(f)
```

Comment:
I wonder if users might be surprised by this also being part of the script behaviour? It should probably be mentioned in the doc-string at least.

Reply:
I've clarified in a local commit, but do note that "users" are only internal codeql-developers right now.